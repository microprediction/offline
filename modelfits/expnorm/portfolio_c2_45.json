{"g1": 0.014873651268125516, "g2": 12.524525362091516, "logK": -4.475570868496839, "loc": 0.032210562700284884, "logScale": 0.8271752204193601}