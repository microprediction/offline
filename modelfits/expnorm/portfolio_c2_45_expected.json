{"g1": 0.7439748887789481, "g2": 3.9312831621920763, "logK": 0.9997632818098176, "loc": -0.11537059656480093, "logScale": -1.8714058774097109}